instrumentation/servlet/servlet-javax-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/javax/JavaxServletAccessor.java

```diff
@@ -96,6 +98,19 @@ public List<String> getRequestParameterValues(
     return values == null ? Collections.emptyList() : Arrays.asList(values);
   }
 
+  @Override
+  @SuppressWarnings("unchecked")
+  public Map<String, String[]> getQueryParamsMap(HttpServletRequest request) {
```

Comment:
Nit: if you wanted to avoid copying all the query params to an extra map instead of `getQueryParamsMap()` you could have a method like `iterateOverQueryParams(HttpServletRequest, BiConsumer<String, String[]>)`